README.md

```diff
@@ -103,6 +103,14 @@ The Upgrader updates projects via one of the following approaches:
 ```bash
 export UPGRADE_STRATEGY=composer
 ```
+#### Composer update strategy
+Composer install|update uses two strategies.
+By default composer updates only lock file.
+
+* To enable update composer with packages:
+```bash
+export COMPOSER_INSTALL_DEPENDENCIES=false
```

Comment:
Is it correct behavior? 

`To enable update composer with packages = COMPOSER_INSTALL_DEPENDENCIES = false`

From my perspective better:

`To enable update composer with packages = COMPOSER_INSTALL_DEPENDENCIES = true`

Can we change the behavior to the second version?

Reply:
it's correct. see comment above

Reply:
Yes you are right 
By default: we have use `--no-install flag 
That's mean: by default, config `COMPOSER_INSTALL_DEPENDENCIES` = false
and for enable it we will add `export COMPOSER_INSTALL_DEPENDENCIES = true`